README.md

```diff
@@ -8,6 +8,8 @@ The Postgres DB runs in a docker container. To Start the DB, run `sh start-datab
 
 Once you have created the DB, you need to duplicate the `.env.example` fil, rename to `.env` and fill in your values.
 
+Run bun db:push
```

Comment:
Add backticks to format the command as code, helps the reader distinguish instruction text from commands/code.

```suggestion
Run `bun db:push`
```